CONTRIBUTING.md

```diff
@@ -194,3 +194,76 @@ IntelliJ may not pick up protoc generated classes as they can be very huge. If t
 
 ## CI
 The Ozone project uses Github Actions for its CI system.  The configuration is described in detail [here](.github/ci.md).
+
+## Debugging
```

Comment:
These changes should be off the master branch, not 1.2.0.   This should be called "Connecting to  Ozone on Kubernetes" and should be below "run ozone from intellij" section above.